io.sloeber.product/neon-cdt-nebula.target

```diff
@@ -44,5 +40,24 @@
 <unit id="org.eclipse.platform.sdk" version="4.6.1.M20160907-1200"/>
 <repository location="http://download.eclipse.org/eclipse/updates/4.6"/>
 </location>
+<location includeAllPlatforms="true" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="ilg.gnuarmeclipse.doc.user.feature.group" version="1.1.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.codered.feature.group" version="1.1.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.debug.gdbjtag.openocd.feature.group" version="4.1.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.debug.gdbjtag.pyocd.feature.group" version="1.1.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.managedbuild.cross.feature.group" version="2.2.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.packs.feature.group" version="2.2.2.201606210758"/>
+<unit id="ilg.gnuarmeclipse.templates.stm.feature.group" version="2.5.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.templates.freescale.feature.group" version="2.2.7.201606210758"/>
+<unit id="ilg.gnuarmeclipse.debug.gdbjtag.jlink.feature.group" version="4.1.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.templates.ad.feature.group" version="1.1.3.201606210758"/>
+<unit id="ilg.gnuarmeclipse.templates.cortexm.feature.group" version="1.3.1.201606210758"/>
+<unit id="ilg.gnuarmeclipse.debug.gdbjtag.qemu.feature.group" version="3.1.1.201606210758"/>
+<repository location="http://sloeber.io/gnuarmeclipse/stable"/>
+</location>
+<location includeAllPlatforms="true" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.nebula.widgets.oscilloscope.feature.feature.group" version="1.2.0.201612121953"/>
+<repository location="http://download.eclipse.org/nebula/snapshot"/>
+</location>
 </locations>
```

Comment:
http://www.vogella.com/tutorials/EclipseTargetPlatform/article.html

Define all plugins needed for compilation as separate from your working Eclipse with JDT and PDE